samples/MvcSample.Web/Startup.cs

```diff
@@ -48,7 +39,6 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
             services.AddMvcLocalization(LanguageViewLocationExpanderFormat.SubFolder);
 
-#if DNX451
             // Fully-qualify configuration path to avoid issues in functional tests. Just "config.json" would be fine
             // but Configuration uses CallContextServiceLocator.Locator.ServiceProvider to get IApplicationEnvironment.
             // Functional tests update that service but not in the static provider.
```

Comment:
Please remove this comment while you're here. It's no longer correct: `ConfigurationBuilder` no longer uses the `static` `IServiceProvider` property and fully-qualifying the `config.json` path is about the same as passing `applicationEnvironment.ApplicationBasePath` to the `ConfigurationBuilder` constructor.